ts/mixins/centered-series.ts

```diff
@@ -88,16 +88,34 @@ H.CenteredSeriesMixin = {
             plotWidth = chart.plotWidth - 2 * slicingRoom,
             plotHeight = chart.plotHeight - 2 * slicingRoom,
             centerOption: Array<(number|string|null)> = options.center as any,
-            positions: Array<number> = [
-                pick(centerOption[0] as any, '50%' as any),
-                pick(centerOption[1] as any, '50%' as any),
-                options.size || '100%' as any,
-                options.innerSize || 0
-            ],
             smallestSize = Math.min(plotWidth, plotHeight),
+            size = options.size,
+            innerSize = options.innerSize,
+            positions: Array<number>,
             i: number,
             value: number;
 
+        if (typeof size === 'string') {
+            size = parseFloat(size);
+        }
+
+        if (typeof innerSize === 'string') {
+            innerSize = parseFloat(innerSize);
+        }
+
+        positions = [
+            pick(centerOption[0] as any, '50%' as any),
+            pick(centerOption[1] as any, '50%' as any),
+            // Prevent from negative values
+            pick(size && size < 0 ? void 0 : options.size, '100%'),
+            pick(innerSize && innerSize < 0 ? void 0 : options.innerSize, '0%')
+        ];
```

Comment:
Can we use `getCenter()` method from cetnered-series mixin?

Reply:
This change is applied inside the `getCenter` function. I have added it in order to prevent from negative values for radiuses in general when generating positions by the `getCenter` function. Right now setting the `pane.size` as some negative results in error (https://jsfiddle.net/BlackLabel/fLxosc71/). What do you think about it? Should I change that to how it worked before?

Reply:
> This change is applied inside the getCenter function.

Oh right, my bad!